src/liquid/components/contact_type_property.html

```diff
@@ -1,17 +1,39 @@
-{% if include.prop.type != 'generated' %}
+<div id="div_{{include.prop.property_name}}">
+  {% if include.prop.type != 'generated' %}
   {% capture prop_name %}{{ include.prefix }}{{include.prop.property_name}}{% endcapture %}
+  {% if include.prop.type == 'dob' %}
```

Comment:
can we make a `contact_type_dob.html` (or similar)?  i suspect that'd be cleaner and can hopefully be used directly by your api endpoint

Reply:
especially with this inbound also https://github.com/medic/cht-user-management/pull/96/files#diff-cffbfbd30f96baa3bf1eecaf18f63d0fce51c820e87c0cf589510bd88bb2ce1eR8

Reply:
don't agree with this? no worries if not.